kolibri_instant_schools_plugin/assets/src/views/profile-page/index.vue

```diff
@@ -150,23 +155,26 @@
     mixins: [responsiveWindow],
     data() {
       return {
-        username: this.session.username,
         name: this.session.full_name,
-        usernameBlurred: false,
-        nameBlurred: false,
         formSubmitted: false,
+        newPw: '',
+        newPwBlurred: false,
+        newPwConfirm: '',
+        newPwConfirmBlurred: false,
       };
     },
     computed: {
       role() {
-        if (this.getUserRole === UserKinds.ADMIN) {
```

Comment:
This is the kind of refactor that @indirectlylit was requesting that we try to avoid - what is the motivation for changing it from an if sequence to a switch?

Reply:
DRYness and readability. I will revert this.